Revise GCZ deployment instructions for AKS

diksha-hop opened the pull request:
Updated deployment instructions for Geospatial Consumption Zone (GCZ) on Azure Kubernetes Service (AKS) with new environment variables and Helm chart configurations.

diksha-hop committed:
Revise GCZ deployment instructions for AKS

Updated deployment instructions for Geospatial Consumption Zone (GCZ) on Azure Kubernetes Service (AKS) with new environment variables and Helm chart configurations.

diksha-hop committed:
Update deployment date for GCZ on AKS guide

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -38,110 +42,141 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
-   $AZURE_CLIENT_SECRET="<clientSecret>" # App Registration client secret. Example: Aa1Bb~2Cc3.-Dd4Ee5Ff6Gg7Hh8Ii9_Jj0Kk1Ll2
-   $SCOPE="<scope>" # Scope of the App Registration. Example: 00001111-aaaa-2222-bbbb-3333cccc4444/.default
-   $CALLBACK_URL="http://localhost:8080" # Redirect URI of the ADME App Registration (from scope) ie: http://localhost:8080
-   $PRIVATE_NETWORK="true" # Set to false if you want to expose the service publicly using a LoadBalancer. You can still expose the service using an Ingress Controller or Azure API Management at a later stage.
```
I think `PRIVATE_NETWORK="true"` is still needed for this official GCZ Microsoft docs.

jorchiu replied:
<img width="1331" height="1433" alt="image" src="https://github.com/user-attachments/assets/f6b5c5c4-cf3b-4068-8855-e94bc6f407de" />

jorchiu replied:
The official docs have changed the helm chart slightly to include private network

diksha-hop replied:
Added them, please review

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -38,110 +42,141 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+   $GCZ_TRANSFORMER_IMAGE_TAG="0.28.2"
+
+   # Istio Configuration (Enable ONLY if Istio exists on AKS)
+   $ISTIO_ENABLED="true"
```
Leave $ISTIO_ENABLED="false" since this tutorial doesn't use istio.

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -38,110 +42,141 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+    service:
+      type: LoadBalancer
+    configuration:
+      privateNetwork: "$PRIVATE_NETWORK"
```
PRIVATE_NETWORK needs to be defined

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -153,51 +188,53 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      gcz_ignite_service: "$GCZ_IGNITE_SERVICE"
    "@ | Out-File -FilePath osdu_gcz_custom_values.yaml
    ```
 
````
Please double check the service files for provider and transformer here.

The port and target ports should match with the community docs 
[provider/templates/service.yaml](https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/devops/azure/charts/provider/templates/service.yaml) and [transformer/templates/service.yaml](https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/devops/azure/charts/transformer/templates/service.yaml)

diksha-hop committed:
Update Istio configuration settings in deployment guide

diksha-hop committed:
Change header level for GCZ deployment guide

Updated header level for the deployment guide.

diksha-hop committed:
Update PowerShell tab link for deploy-gcz instructions

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -38,112 +42,145 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+EOF
+```
+
+   ### [Windows PowerShell](#tab/deploy-gcz)
````
Does this need to be changed? Could we keep it as `### [Windows PowerShell](#tab/windows-powershell)`

diksha-hop replied:
its giving me this error again and again 
<img width="639" height="237" alt="image" src="https://github.com/user-attachments/assets/4fb8fe3f-281f-44b0-9779-ae7fd0854994" />

ShannonLeavitt replied:
@diksha-hop - To resolve this error, try adding a closing `---` to the end of the set of tabs. A tab set can contain only uniquely named tabs.

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -153,51 +190,53 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
-               clientSecret: $(echo "$AZURE_CLIENT_SECRET" | base64)
-               gcz_ignite_namespace: "$GCZ_IGNITE_NAMESPACE"
-               gcz_ignite_service: "$GCZ_IGNITE_SERVICE"
+  ignite:
```
Please fix the indentation for this yaml.

diksha-hop replied:
fixed it

diksha-hop committed:
Fix heading link for Windows PowerShell section

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -38,110 +42,143 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+   $VERSION="0.28.2"
+```
 
 1. Create the HELM chart:
````
The markdown formatting is off here. 

https://github.com/diksha-hop/azure-docs/blob/patch-2/articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md

<img width="877" height="1140" alt="image" src="https://github.com/user-attachments/assets/be915b09-4441-4ea8-9855-9a25962e3c4e" />

diksha-hop committed:
Update GCZ deployment configuration on AKS

Updated the deployment configuration for GCZ on AKS by modifying the Helm chart values and ensuring the service type is set to LoadBalancer for provider and transformer services. Adjusted the configuration file to include necessary parameters and instructions for deployment.

diksha-hop committed:
Update Unix Shell tab reference in deployment guide

diksha-hop committed:
Update deploy-gcz-on-aks.md

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -316,7 +363,8 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
-   helm upgrade -i $CHART . -n $NAMESPACE -f osdu_gcz_custom_values.yaml --set-file global.provider.configLoaderJs="../../../../gcz-provider/gcz-provider-core/config/configLoader.js"
+   helm upgrade -i "$CHART" . -n "$NAMESPACE" -f osdu_gcz_custom_values.yaml \
+     --set-file global.provider.configLoaderJs="../../../../gcz-provider/gcz-provider-core/config/configLoader.js"
    ```
````
I think we should also add steps 7-11 from the community docs
<img width="1392" height="711" alt="image" src="https://github.com/user-attachments/assets/f8eed0aa-c8cc-49a8-b85f-450788f4705f" />
https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/devops/azure/charts/geospatial/README.md

diksha-hop committed:
Update deploy-gcz-on-aks.md

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -332,6 +380,28 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
    ```bash
    kubectl get service -n $NAMESPACE
    ```
+1. Port-forward to the GCZ Provider service and test the FeatureServer layers:
````
Please use the description of the step's verbatim from the community for each step below in your changes

For example, instead of `Port-forward to the GCZ Provider service and test the FeatureServer layers:`, change this to `Test the gcz-provider endpoint by port forwarding. In the curl command, the response should contain the layer definitions under [koop-config.json](https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/provider/gcz-transformer-azure/src/main/resources/koop-config.json)`

<img width="1392" height="711" alt="image" src="https://github.com/user-attachments/assets/1813cee3-673c-4b6a-92ed-e13694d38abe" />

diksha-hop replied:
made these changes

diksha-hop committed:
Update deploy-gcz-on-aks.md

diksha-hop commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
@microsoft-github-policy-service agree company="Microsoft"

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -89,57 +135,56 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+         crsconvertorURL: "https://$AZURE_DNS_NAME/api/crs/converter/v3/convertTrajectory"
+         storageURL: "https://$AZURE_DNS_NAME/api/storage/v2/records"
+         clientSecret: $(echo "$AZURE_CLIENT_SECRET" | base64)
+         gcz_ignite_namespace: "$GCZ_IGNITE_NAMESPACE"
```
Please double check the [configuration from the community] for transformer, provider, and ignite (https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/devops/azure/charts/geospatial/README.md#create-the-gcz-helm-chart-values), and add the missing properties such as `gcz_persistence_enabled`.

jorchiu replied:
<img width="1194" height="737" alt="image" src="https://github.com/user-attachments/assets/fbaa3d5f-1cd8-43c4-970b-38a804ff1d8c" />

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -89,57 +135,56 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+       namespace: $NAMESPACE
+       entitlementsGroupsURL: "https://$AZURE_DNS_NAME/api/entitlements/v2/groups"
+       image:
+         repository: community.opengroup.org:5555
```
The properties under image should come from the values. Similar to what's been done in the [community](https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/devops/azure/charts/geospatial/README.md#create-the-gcz-helm-chart-values)

<img width="1087" height="398" alt="image" src="https://github.com/user-attachments/assets/fa95051c-72bf-457c-a832-f575f04098c3" />

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -89,57 +135,56 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+       image:
+         name: gridgain/community
+         tag: 8.8.43
+       configuration:
```
This `ignite` section is missing additional settings. Please review what's in the [community](https://community.opengroup.org/osdu/platform/consumption/geospatial/-/blob/master/devops/azure/charts/geospatial/README.md?ref_type=heads#create-the-gcz-helm-chart-values).


<img width="661" height="873" alt="image" src="https://github.com/user-attachments/assets/b38b9ad0-2f10-4c0f-a23d-69553e17854b" />


<img width="605" height="267" alt="image" src="https://github.com/user-attachments/assets/669c33cd-dbeb-401b-894d-bb2b6b547b52" />

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -153,51 +198,53 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
-               clientSecret: $(echo "$AZURE_CLIENT_SECRET" | base64)
-               gcz_ignite_namespace: "$GCZ_IGNITE_NAMESPACE"
-               gcz_ignite_service: "$GCZ_IGNITE_SERVICE"
+     ignite:
```
Same comment as before https://github.com/MicrosoftDocs/azure-docs/pull/127798/files#r2565747179

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -153,51 +198,53 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+       configuration:
+         privateNetwork: "$PRIVATE_NETWORK"
+
+     transformer:
```
Same comment as before https://github.com/MicrosoftDocs/azure-docs/pull/127798/files#r2565722544

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -89,57 +135,56 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+     transformer:
+       namespace: $NAMESPACE
+       image:
+         repository: community.opengroup.org:5555
```
Same comment as above https://github.com/MicrosoftDocs/azure-docs/pull/127798/files#r2565727808

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -153,51 +198,53 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+     transformer:
+       namespace: $NAMESPACE
+       image:
+         repository: community.opengroup.org:5555
```
Same comment as before https://github.com/MicrosoftDocs/azure-docs/pull/127798/files#r2565727808

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -153,51 +198,53 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+       namespace: $NAMESPACE
+       entitlementsGroupsURL: "https://$AZURE_DNS_NAME/api/entitlements/v2/groups"
+       image:
+         repository: community.opengroup.org:5555
```
Same comment as before https://github.com/MicrosoftDocs/azure-docs/pull/127798/files#r2565727808

diksha-hop replied:
@jorchiu I have made all the suggested changes as above, please take a look

diksha-hop committed:
fixing the suggested changes

diksha-hop committed:
Refactor GCZ Azure deployment configuration

Updated GCZ configuration for Azure deployment, including changes to persistence folder and image repository settings.

diksha-hop committed:
Fix formatting in deploy-gcz-on-aks.mdf

diksha-hop committed:
Add service.yaml template for AKS

Added service.yaml template for AKS deployment.

diksha-hop committed:
Fix formatting in deploy-gcz-on-aks.md

Removed an unnecessary line break before the shell command for creating the HELM chart.

diksha-hop committed:
Fix tab references in AKS deployment guide

Updated tab references for Unix Shell and Windows PowerShell sections in the deployment guide.

diksha-hop committed:
Refactor deployment instructions for GCZ on AKS

diksha-hop committed:
Fix formatting in deploy-gcz-on-aks.md

diksha-hop committed:
Update step numbers in deployment guide

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,313 +31,395 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      storageURL: "https://$AZURE_DNS_NAME/api/storage/v2/records"
+      clientSecret: [Convert]::ToBase64String([Text.Encoding]::UTF8.GetBytes($AZURE_CLIENT_SECRET))
+      gcz_persistence_enabled: true
+      gcz_persistence_folder: "/persistence/storage"
```
Is this `gcz_persistence_folder` needed?

diksha-hop committed:
Remove gcz_persistence_folder from deployment config

Removed gcz_persistence_folder configuration from YAML files while keeping gcz_persistence_enabled set to true.

diksha-hop committed:
Merge branch 'main' into patch-2

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +31,393 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      type: LoadBalancer
+    configuration:
+      privateNetwork: "$PRIVATE_NETWORK"
+      datapartitionid: $DATA_PARTITION_ID
```
Should be `dataPartitionId`

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +31,393 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      crsconvertorURL: "https://$AZURE_DNS_NAME/api/crs/converter/v3/convertTrajectory"
+      storageURL: "https://$AZURE_DNS_NAME/api/storage/v2/records"
+      clientSecret: $(echo "$AZURE_CLIENT_SECRET" | base64)
+      gcz_persistence_enabled: true      
```
Missing azureAppResourceId and gcz_ignite_service

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +31,393 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      type: LoadBalancer
+    configuration:
+      privateNetwork: "$PRIVATE_NETWORK"
+      datapartitionid: $DATA_PARTITION_ID
```
Should be `dataPartitionId`

jorchiu commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +31,393 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      crsconvertorURL: "https://$AZURE_DNS_NAME/api/crs/converter/v3/convertTrajectory"
+      storageURL: "https://$AZURE_DNS_NAME/api/storage/v2/records"
+      clientSecret: [Convert]::ToBase64String([Text.Encoding]::UTF8.GetBytes($AZURE_CLIENT_SECRET))
+      gcz_persistence_enabled: true      
```
Missing azureAppResourceId and gcz_ignite_service

diksha-hop committed:
Fix casing of dataPartitionId in AKS deployment guide

Corrected the casing of 'datapartitionid' to 'dataPartitionId' in configuration sections.

diksha-hop committed:
Add azureAppResourceId and gcz_ignite_service

diksha-hop committed:
Refactor GCZ deployment configuration on AKS

Updated deployment configuration for GCZ on AKS, including image repository and tag changes for provider, transformer, and ignite services. Adjusted service types to LoadBalancer and modified YAML formatting.

diksha-hop committed:
Update header level for GCZ deployment guide

diksha-hop committed:
Revise GCZ deployment instructions for AKS

Updated the deployment guide for Geospatial Consumption Zone (GCZ) on Azure Kubernetes Service (AKS) to include environment variable exports, service type changes, and Helm chart deployment instructions.

EirikHaughom commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -11,9 +11,13 @@ ms.date: 05/30/2024
+This guide explains how to deploy Geospatial Consumption Zone (GCZ) as an **add-on extensibility service** for Azure Data Manager for Energy (ADME), using a **Helm chart** running on **Azure Kubernetes Service (AKS)**.
 
-## Prerequisites
+> ✅ Microsoft recommends deploying GCZ on a **dedicated AKS cluster** separate from ADME core services for optimal performance and isolation.
```
Since ADME is deployed in AME tenant, the customer does not even have access to deploy GCZ into the ADME clusters. We should remove this ambiguity from the documentation.

diksha-hop replied:
removed it, thanks for noticing.

EirikHaughom commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -27,314 +31,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
    ```
 
-   Now you should see the pods for the `ignite`, `provider`, and `transformer` services.
+   Now you should see the pods for the `ignite`, `provider`, gridgain, and `transformer` services.
````
`gridgain` to ensure consistency.

diksha-hop replied:
fixed it

diksha-hop committed:
Revise GCZ deployment instructions for AKS

Updated the deployment guide for GCZ on AKS to improve clarity and consistency.

diksha-hop committed:
Update GCZ deployment instructions on AKS

Removed recommendation for dedicated AKS cluster for GCZ deployment.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+   $AZURE_TENANT_ID="<TENANT_ID_of_target_OSDU_deployment>"  # Entra ID tenant ID. Example: aaaabbbb-0000-cccc-1111-dddd2222eeee
+   $AZURE_CLIENT_ID="<CLIENT_ID_of_target_OSDU_deployment>"  # App Registration client ID. Example: 00001111-aaaa-2222-bbbb-3333cccc4444
+   $AZURE_CLIENT_SECRET="<CLIENT_SECRET_of_target_OSDU_deployment>"  # App Registration client secret. Example: Aa1Bb~2Cc3.-
+   $CLIENT_SECRET_B64=$(echo -n "$CLIENT_SECRET" | base64 -w0)
```
Not applicable. OK to ignore this suggestion.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+ ### [Unix Shell](#tab/unix-shell-1)
+ 
+ ```bash
+$ cat > osdu_gcz_custom_values.yaml << EOF
````
Not applicable. OK to ignore this suggestion.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
-   EOF
-
-   cat > ../transformer/templates/service.yaml << EOF
+   $ cat > ../transformer/templates/service.yaml << EOF
```
Not applicable. OK to ignore this suggestion.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
````diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+
+   ```bash
+   kubectl create secret generic client-secret -n ignite \
+  --from-literal=clientSecret="$CLIENT_SECRET"
````
Not applicable. OK to ignore this suggestion.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
-               clientSecret: $(echo "$AZURE_CLIENT_SECRET" | base64)
-               gcz_ignite_namespace: "$GCZ_IGNITE_NAMESPACE"
-               gcz_ignite_service: "$GCZ_IGNITE_SERVICE"
+   $ cat > ../transformer/templates/service.yaml << EOF
```
Not applicable. OK to ignore this suggestion.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+      targetPort: 8080
+    configuration:
+      secretName: gcz-client-secret
+      configuration:
```
Not applicable. OK to ignore this suggestion.

v-dirichards commented on articles/energy-data-services/includes/how-to/how-to-deploy-gcz/deploy-gcz-on-aks.md:
```diff
@@ -27,314 +29,441 @@ Learn how to deploy Geospatial Consumption Zone (GCZ) on Azure Kubernetes Servic
+   $GCZ_PROVIDER_IMAGE_TAG="0.28.2"
+   $GCZ_TRANSFORMER_IMAGE_NAME="geospatial-transformer"
+   $GCZ_TRANSFORMER_IMAGE_TAG="0.28.2"
+   PROVIDER_IMAGE_REPO=myregistry.azurecr.io/provider
+   PROVIDER_IMAGE_NAME=gcz-provider
+   PROVIDER_IMAGE_TAG=v1.0.0
+   IGNITE_IMAGE_REPO=myregistry.azurecr.io/gridgain
+   IGNITE_IMAGE_NAME=ignite
+   IGNITE_IMAGE_TAG=8.9.11
+   TRANSFORMER_IMAGE_REPO=myregistry.azurecr.io/transformer
+   TRANSFORMER_IMAGE_NAME=gcz-transformer
+   TRANSFORMER_IMAGE_TAG=v1.0.0
```
Not applicable. OK to ignore this suggestion.

Court72 committed:
Fix issues found in the staged PR
